### ISSUE TITLE
Change default message size to something greater than 2048

### ISSUE THREAD

**Issue by mrsnyds** [home-assistant/home-assistant.github.io#27841]
### Feedback

I have a situation where I need to increase the default text size limit, and cannot find instructions for doing this in the documentation.  

I read ... 

`If the default maximum message size (2048 bytes) to be used in events is too small for your needs, then this maximum size setting can be increased. You need to have your profile set to advanced mode to do this.`

... but cannot find the steps to increase message size?

Also, and this may be a code "issue", I read that this size should not be limited when I use the text attribute as a variable in a template.   I have attempted to do this by assigning the attribute to a variable, and writing it to the logbook, but it appears the entry as read in the logbook is still truncated at 2048.   So it seems I need to know how to increase the default size.

### URL

https://www.home-assistant.io/integrations/imap/

### Version

2023.6.2

**Comment by jbouwh** [comment]
Did you enable advanced mode in your personal profile as required?

**Comment by jbouwh** [comment]
![image](https://github.com/home-assistant/home-assistant.io/assets/7188918/2d816a0a-316e-4270-9a22-3a93f6af8ae6)
 You can also use the new built-in custom template. This new feature does not have this limit.

We limit the event event size to because for performance reasons.

**Comment by mrsnyds** [comment]
Thanks for the quick reply.

Yes, I did have advanced mode turned on.  I'm usually pretty good at poking around and finding things, but I just didn't think to look at the configure button on the integrations page (duh!!).  Perhaps an short edit to the doc' to tell readers where to look would be helpful.?  I was able to increase the size modestly (5000) and capture enough of the text for my purposes, but I would switch to the custom template approach if there is a worthwhile performance difference (I'm questioning that because it looks like it is dependent on the same event trigger).

![image](https://github.com/home-assistant/home-assistant.io/assets/41122989/235980fb-0255-4cee-8683-60db712335c1)
